### DISCUSSION TITLE
How to ignore SQL scripts outside `db/views`?

### DISCUSSION THREAD

**Discussion by onwardmk** [scenic-views/scenic#391] [Q&A] [answered]
I have some existing SQL scripts in `db/sql` that create views.

When I run `rails g scenic:view my_view`, scenic finds those scripts and then updates my `schema.rb` file to include their contents. Is there something I can do to prevent this behavior?

**Comment by derekprior** [comment, marked as answer]
The scenic schema dumper dumps all views in the database that is configured for your current environment. This is consistent with how the rails schema dumper handles tables. If you had scripts that created tables then ran migrations, you would find those tables in your schema too.

I have two pieces of advice:

1. Just let Scenic manage all of your views
2. If that won't work for you, you should be able to use [`ignore_tables`](https://blog.saeloun.com/2023/01/10/rails-add-ability-to-ignore-tables-by-regexp/) in order to prevent certain views from being dumped.

**Reply by onwardmk** [reply]
Thank you for the suggestions!

**Reply by beniamin1213a-cloud** [reply]
awli

**Reply by beniamin1213a-cloud** [reply]
parcham

**Reply by skamir2794gk-collab** [reply]
e.g., https://opencollective.com/babel<---^|

**Comment by Duge6715** [comment]
Thanks for suggestions!
